### PR TITLE
os/arch/xtensa: change argment of function board_app_initialize() in defination;

### PR DESCRIPTION
**Pull request by glb1013**
make sure function board_app_initialize() be same with its prototype in os\include\tinyara\board.h;

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [f834301dc07b8884d3652e3e06318f46557eb675] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [f834301dc07b8884d3652e3e06318f46557eb675] - Code Rule Check OK.

File: os/board/esp32_devkit/src/esp32_appinit.c [hunk 1]
```diff
@@ -94,7 +94,7 @@
  *
  ****************************************************************************/
 
-int board_app_initialize(uintptr_t arg)
```
Comment (xixidodo) [inline os/board/esp32_devkit/src/esp32_appinit.c]: please also change this function Description comment accordingly.

Comment (glb1013) [reply os/board/esp32_devkit/src/esp32_appinit.c]: Modified.

**Comment by seinfra** [comment]
Target : [bca39c9678c4d6366aa5069cb01bcff7ad78640f] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [bca39c9678c4d6366aa5069cb01bcff7ad78640f] - Code Rule Check OK.

**Comment by glb1013** [comment]
> Please add the following information at the end of your commit comment.
> Signed-off-by: glb1013 [longbao.guo@samsung.com](mailto:longbao.guo@samsung.com)

Done

**Comment by seinfra** [comment]
Target : [4fafe81c7826d8d937fc0cde345fc859f556dafe] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [4fafe81c7826d8d937fc0cde345fc859f556dafe] - Code Rule Check (C++) OK.

File: os/board/esp32_devkit/src/esp32_appinit.c [hunk 2]
```diff
@@ -77,24 +77,9 @@
...
- *         implementation without modification.  The argument has no
- *         meaning to NuttX; the meaning of the argument is a contract
- *         between the board-specific initalization logic and the
- *         matching application logic.  The value cold be such things as a
- *         mode enumeration value, a set of DIP switch switch settings, a
- *         pointer to configuration data read from a file or serial FLASH,
- *         or whatever you would like to do with it.  Every implementation
- *         should accept zero/NULL as a default configuration.
- *
- * Returned Value:
```
Comment (sunghan-chang) [inline os/board/esp32_devkit/src/esp32_appinit.c]: Why do you remove "returned value"?

Comment (glb1013) [reply os/board/esp32_devkit/src/esp32_appinit.c]: It has no return value, and keep it same with other implements' comments.

**Comment by seinfra** [comment]
Target : [fba0d2e43e540efd465ab2ad0e60d6faa7854243] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [fba0d2e43e540efd465ab2ad0e60d6faa7854243] - Code Rule Check (C++) OK.

File: os/board/esp32_devkit/src/esp32_appinit.c (hunk 2 above)
Comment (glb1013) [reply os/board/esp32_devkit/src/esp32_appinit.c]: Modified.

**Pull request merged**